### PR TITLE
Get started in your notebook

### PR DESCRIPTION
**Pull request by ivarpruijn**

### CODE REVIEW

File: bach/docs/source/get-started-in-your-notebook.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		except Exception as e:
+			os.remove(pk_path)
+			raise(e)
+
+5) Setup the db connection
+
+.. code-block:: python
+
+	port = connect_tunnel(ssh_host='SSH_HOSTNAME', db_port=PORT)
+	df = modelhub.get_objectiv_dataframe(db_url=f'postgresql://USER:PASSWORD@localhost:{port}/DATABASE,
```
Comment (borft) [inline bach/docs/source/get-started-in-your-notebook.rst:107]:
```suggestion
	df = modelhub.get_objectiv_dataframe(db_url=f'postgresql://USER:PASSWORD@localhost:{port}/DATABASE,
```

File: bach/docs/source/get-started-in-your-notebook.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+	
+	pip install sshtunnel
+
+	from sshtunnel import SSHTunnelForwarder
+	import os, stat
+
+	def connect_tunnel(ssh_host: str, db_host: str = 'localhost', db_port: int = PORT) -> int:    
+		try:
+
+			ssh_username= SSH_USER
```
Comment (borft) [inline bach/docs/source/get-started-in-your-notebook.rst:78]: We could use the predefined secrets directly, as we only use them once anyway, that would save a few lines of code here. Additionally, we should probably explain that `PORT`, `SSH_USER`, `SSH_PASSWORD` and `SSH_KEY` are expected to be set, prior to calling this method. And what they need to be set to.

Comment (borft) [reply bach/docs/source/get-started-in-your-notebook.rst:78]: perhaps we should also add a docstring here

File: bach/docs/source/get-started-in-your-notebook.rst [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+	from modelhub import ModelHub
+
+3) Instantiate the model hub and set the default time aggregation:
+
+.. code-block:: python
+
+	modelhub = ModelHub(time_aggregation='%Y-%m-%d')
+
+4) Optionally: create an SSH tunnel:
```
Comment (borft) [inline bach/docs/source/get-started-in-your-notebook.rst:66]:
```suggestion
4) Optionally: create an SSH tunnel to the Postgres database server:

```

**Review by borft — COMMENTED** [review]
a few minor suggestions

File: bach/docs/source/get-started-in-your-notebook.rst [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+		except Exception as e:
+			os.remove(pk_path)
+			raise(e)
+
+5) Setup the db connection
+
+.. code-block:: python
+
+	port = connect_tunnel(ssh_host='SSH_HOSTNAME', db_port=PORT)
```
Comment (borft) [inline bach/docs/source/get-started-in-your-notebook.rst:106]:
```suggestion
	port = connect_tunnel(ssh_host=SSH_HOSTNAME, db_port=PORT)
```

File: bach/docs/source/get-started-in-your-notebook.rst [hunk 5]
```diff
@@ -70,45 +70,48 @@ Google Colab / Hex / Deepnote
 	
 	pip install sshtunnel
```
Comment (vincenthoogsteder) [inline bach/docs/source/get-started-in-your-notebook.rst]: @borft moet hier en tab voor?

**Review by vincenthoogsteder — APPROVED** [review]
LGTM

**Pull request merged**